src/qiskit_iqm/fake_backends/chip_sample.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2022-2023 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-instance-attributes
+"""Abstract representation of an IQM chip sample.
+"""
+from dataclasses import dataclass
+from typing import Any, Union
+
+from iqm_client import QuantumArchitectureSpecification
+
+
+@dataclass
+class IQMChipSample:
+    """
+    Physical properties of a quantum chip sample.
+
+    Args:
+        quantum_architecture: Quantum architecture specification of the chip.
+        t1s: :math:`T_1` times (in ns) for each qubit of the chip, corresponding key is the physical qubit name.
+        t2s: :math:`T_2` times (in ns) for each qubit of the chip, corresponding key is the physical qubit name.
+        single_qubit_gate_depolarizing_error_parameters: Depolarizing error parameters for single-qubit gates of each
+            qubit. Using the values in the depolarizing channel, concatenated with a thermal relaxation channel,
+            lead to average gate fidelities that would be determined by benchmarking.
+        two_qubit_gate_depolarizing_error_parameters: Depolarizing error parameters for two-qubit gates of each
+            connection. Using the values in the depolarizing channel, concatenated with a thermal relaxation channel,
+            lead to average gate fidelities that would be determined by benchmarking.
+        single_qubit_gate_durations: Gate duration (in ns) for each single-qubit gate
+        two_qubit_gate_durations: Gate duration (in ns) for each two-qubit gate.
+        id_: Identifier of the chip sample. Defaults to None.
+
+    Example:
+        .. code-block::
+
+            IQMChipSample(quantum_architecture=ThreeQubitExample(),
```

Comment:
`ThreeQubitExample` is not something that is defined anywhere. I suggest just to create a quantum architecture from scratch in-place here